src/proto/gps.proto

```diff
@@ -20,6 +20,9 @@ enum SampleType {
   TOTAL_DATA_TYPES = 13;
   POSITION = 14;
   LINEAR_VELOCITY = 15;
+  RGB_IMAGE_SIZE = 16;
+  DOMAIN_IMAGE = 17;
```

Comment:
Yes, exactly.
On Apr 18, 2016 12:26 PM, "Bradly Stadie" <notifications@github.com> wrote:

> In src/proto/gps.proto
> <https://github.com/cbfinn/gps/pull/15#discussion_r60119186>:
>
> > @@ -20,6 +20,9 @@ enum SampleType {
> >    TOTAL_DATA_TYPES = 13;
> >    POSITION = 14;
> >    LINEAR_VELOCITY = 15;
> > +  RGB_IMAGE_SIZE = 16;
> > +  DOMAIN_IMAGE = 17;
>
> I see. So for example if you have a maze and you want to show the agent
> the maze at the beginning of the trial?
>
> @cbfinn <https://github.com/cbfinn> can you think of a concise way to
> word this for the comment?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/cbfinn/gps/pull/15/files/7cb072155dd03779a3cdcaca47d1c59ef5ba1449#r60119186>
>